stdlib/_curses.pyi

```diff
@@ -278,9 +278,9 @@ if sys.platform != "win32":
     def color_content(__color_number: int) -> tuple[int, int, int]: ...
     # Changed in Python 3.8.8 and 3.9.2
     if sys.version_info >= (3, 8):
-        def color_pair(pair_number: int) -> int: ...
+        def color_pair(__pair_number: int) -> int: ...
     else:
-        def color_pair(__color_number: int) -> int: ...
+        def color_pair(pair_number: int) -> int: ...
```

Comment:
I don't have a linux or mac to test on (and Windows doesn't have `curses`), but it looks to me like this was positional-only on 3.7 as well: https://tio.run/##K6gsycjPM/7/PzO3IL@oRKG4spiroCgzr0RDCcjUK0stKs7Mz1PILFbSUUAS0OSCqk8uLSpOLeaCUHrJ@Tn5RfEFiZlFGiAiPq80Nym1yNbESPP/fwA